core/model/src/main/java/it/unibz/inf/ontop/dbschema/impl/DatabaseTableDefinition.java

```diff
@@ -1,18 +1,22 @@
 package it.unibz.inf.ontop.dbschema.impl;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.google.common.collect.ImmutableList;
 import it.unibz.inf.ontop.dbschema.Attribute;
+import it.unibz.inf.ontop.dbschema.DatabaseRelationDefinition;
 import it.unibz.inf.ontop.dbschema.RelationID;
 
 import java.util.stream.Collectors;
-
 public class DatabaseTableDefinition extends AbstractDatabaseRelationDefinition {
 
     /**
      *
      * @param allIds
      * @param builder
      */
+
```

Comment:
Not sure the changes are needed here. Please consider reverting all changes in that file.